src/styles.css

```diff
@@ -0,0 +1,22 @@
+.appLayoutGrowChildFlex { display: -webkit-box !important; display: -webkit-flex !important; display: -ms-flexbox !important; display: flex !important; }
```

Comment:
would be nice since now we are reworking the classes, to use a better class name, `app*` made sense in the past, now that this is an isolated library it would be good to call it something like `reactFlexboxLayout*`?, @byronmwong can you validate.